src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1

```diff
@@ -168,4 +168,33 @@ function Test-NegativeAnalysisServicesServer
 		Invoke-HandledCmdlet -Command {Remove-AzureRmAnalysisServicesServer -ResourceGroupName $resourceGroupName -Name $serverName -ErrorAction SilentlyContinue} -IgnoreFailures
 		Invoke-HandledCmdlet -Command {Remove-AzureRmResourceGroup -Name $resourceGroupName -ErrorAction SilentlyContinue} -IgnoreFailures
 	}
+}
+
+
+function Test-AnalysisServicesServerRestart
+{
+    param
+	(
+		$environment = "aspaaswestusloop1.asazure-int.windows.net",
```

Comment:
Not sure what you mean here. This is just a default value to the test here. The user will provide this an input to the cmdlet.